apps/web/components/Organization/Wrapper/Themes/UniversityStudiesTheme/UniversityStudiesHeader.tsx

```diff
@@ -47,6 +50,14 @@ const UniversityStudiesHeader: React.FC<
 
   const screenWidth = getScreenWidthString(width)
 
+  const { activeLocale } = useI18n()
+  const logoAltText = n(
```

Comment:
Just in case there isn't a title for the organization perhaps it'd be better to fallback to an empty string instead of potentially concatenating undefined or '' with ' Frontpage'